### ISSUE TITLE
phenotype_annotation.tab and phenotype.hpoa are not compatible

### ISSUE THREAD

**Issue by Ju-Yuan** [obophenotype/human-phenotype-ontology#7199]
I found that there are some HPO_term existed in phenotype.hpoa but not in phenotype_annotation.tab, vice versa.
eg. In OMIM:618468 database, HP:0030891 is annotated in phenotype_annotation.tab but not annotated in phenotype.hpoa nor in the HPO website.
In OMIM:615685 database, HP:0001257, HP:0009830 are annotated in phenotype.hpoa but not annotated in phenotype_annotation.tab.

I have read the content in your website, it say"These files were merged into a single file that has been called phenotype_annotation.tab. Starting in 2018, the HPO team is migrating to a new big file format called phenotype.hpoa."
However, it seems like the gene_to_phenotype.txt's data is still based on phenotype_annotation.tab not phenotype.hpoa. eg. I have counted that there are 12,511 database_id in phenotype_annotation.tab which totally included gene_to_phenotype.txt(8,074 database_id), but phenotype.hpoa is not included all the database_id in gene_to_phenotype.txt(only 8,058 database_id).
Based on my observation, the phenotype.hpoa offers the HPO website's data, and phenotype_annotation.tab offers gene_to_phenotype.txt's data(Am I right??). I wonder which file is more accuracy, and what the cause difference between phenotype.hpoa and phenotype_annotation.tab since they have least 10,000 different database_id-HPO_terms.

Looking forward to receiving your reply
Yuan

![圖片 1](https://user-images.githubusercontent.com/56989983/140277690-b1bcf49b-e170-47c7-8baa-9811a34d2223.png)

**Comment by pnrobinson** [comment]
I cannot reproduce this error
```
grep 618468 phenotype.hpoa | grep 0030891
OMIM:618468	Epileptic encephalopathy, early infantile, 76		HP:0030891	PMID:31031012	PCS		2/11			P	HPO:probinson[2020-05-15]
```
I am wondering if maybe you are using different versions of the various resources, since we are adding to the files that are used to create ``phenotype.hpoa`` all the time.  Can you specify what versions of these files you are using?

**Comment by pnrobinson** [comment]
I think possibly this issue stems from "What is the difference between genes_to_phenotypes.txt and phenotype_to_genes.txt ?" (see https://hpo.jax.org/app/faq). Please reopen if needed. The phenotype.hpoa should be considered the source of truth.
